source/globalCommands.py

```diff
@@ -264,6 +264,46 @@ def script_toggleRightMouseButton(self,gesture):
 		else:
 			mouseHandler.lockRightMouseButton()
 
+	@script(
+		description=_(
+			# Translators: Input help mode message for mouse wheel scroll up command.
+			"Scrolls up the mouse wheel at the current mouse position"

```

Comment:
Yes I think this terminology is better for these descriptions

Reply:
Do you mean I don't need to make changes or should I take @CyrilleB79's suggestion?

Reply:
@cary-rowen, if you are concerned by the fact that the function does not move the content directly, do not use scroll at all, because the function does not actually perform scrolling, it actually simulate wheel rotating.

```suggestion
			"Rotates up the mouse wheel at the current mouse position"
```

IMO, no need to mention "simulation", as done also for left/right mouse click. Indeed, assigning a keyboard or touch gestures already makes clear that the real action is not done by the physical mouse.

Reply:
@Qchristensen - what are your thought on the changes from scrolling to rotating? I prefer scrolling and think its more common. Rotating to me seems to imply physical movement of the wheel, not the behaviour on screen.

Reply:
I agree that the verb "to scroll" is more common and probably more understandable.

Actually, my concern with the initial wording was that "to scroll up the mouse wheel" was not correct grammatically. Why not things like:
* "scroll up at the mouse position"  for descriptions / comments
* "mouseScrollUp" for the script's name?

No matter if we do not mention the word "wheel", which create confusion or bad phrasing when used with "scroll".

Reply:
> No matter if we do not mention the word "wheel", which create confusion or bad phrasing when used with "scroll".

I don't entirely agree with this.

I have attached a Microsoft document before. Can we refer to some of the wording in it?

https://support.microsoft.com/en-us/topic/what-are-the-default-buttons-and-wheel-assignments-for-my-mouse-47f656a8-72d0-63ec-8d1d-dda7122bef6b

Reply:
But "scrolling" is actually a key word present in the documentation you linked (first table, in the column "Action")

IMO, when possible, it's better to mention an action rather than a physical button, because there may be other ways to perform them.
For example:
* use "scroll up/down" rather than "rotate the mouse wheel up/down" because there may be other way to perform these actions, e.g. with a touch pad.
* use "scroll left/right" rather than "rotate the mouse wheel left/right because in Chrome you can scroll horizontally using shift + rotating vertically the mouse wheel.